### ISSUE TITLE
[JS][Renderer] Unnecessary `aria-label` on `ac-pushButton`

### ISSUE THREAD

**Issue by compulim** [microsoft/AdaptiveCards#3916]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* JavaScript

# Author or host

We are Bot Framework Web Chat.

# Version of SDK

Web Chat 4.8.0 is using Adaptive Cards 1.2.5, https://github.com/microsoft/BotFramework-WebChat/blob/v4.8.0/packages/bundle/package.json.

# Details

This is an accessibility bug filed to Web Chat and should be moved to AC. I am copying the bug content here. The original bug is at https://github.com/microsoft/BotFramework-WebChat/issues/3090.

![image](https://user-images.githubusercontent.com/1005609/78629201-32e57400-784b-11ea-8a47-f5b07992fda4.png)

**Actual**
aria-label is un-necessarily added in markup

**Expected**
As inner text highlighted in “green” is present, aria-label is not needed.

**WCAG Violation**
Parsing 4.1.1
https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html

**Comment by shalinijoshi19** [comment]
@dclaux initial thoughts on this  ?

**Comment by dclaux** [comment]
While the aria-label is indeed unnecessary, what adverse effect is there to having it there anyway? In other words, does this actually causes a degraded accessibility experience?

**Comment by shalinijoshi19** [comment]
FYI that this code was added specifically to aid Accessibility for outlook WebApp. Need to ensure that any changes here don't end up breaking them. @compulim could you help clarify how much of a blocker this bug is (see also @dclaux's comment above).

**Comment by rp-dev** [comment]
Per customer " It is P4/Low.  There is no user impact. The mark up does not look clean, hence it was reported.  It is OK to not fix this. "

Adaptive card team can decide whether it needs to be fixed or not. Thanks.
